docs/source/user/faq.rst

```diff
@@ -150,6 +150,17 @@ In this case, vectorization is rejected because the vectorized code may behave
 differently.  This is a case to try turning on ``fastmath=True`` to allow
 fastmath instructions.
 
+Why are the ``typed`` containers slower when used from the interpreter?
+-----------------------------------------------------------------------
+
+The Numba ``typed`` containers found in ``numba.typed`` e.g.
+``numba.typed.List`` store their data in an efficient form for access from JIT
+compiled code. When these containers are used from the CPython interpreter, the
```

Comment:
Should this be 2 sentences? i.e. `... from/to the container format. This process is ...`

Reply:
Good point :)

Reply:
b2f68ba  fixes